snapcraft.yaml

```
@@ -102,16 +104,29 @@ parts:
     override-pull: |
       craftctl default
       ./scripts/install/linux_optional_compilation_dependencies.sh
+      ./scripts/install/linux_runtime_dependencies.sh
+      pip3 install --no-input aqtinstall
+      export PATH="$PATH:$HOME/.local/bin"
+      sudo apt update
+      sudo apt install -y lsb-release python3-distro
+      sudo apt install python3-distro
+      pip3 install distro
```

```suggestion
      ./scripts/install/linux_runtime_dependencies.sh
      pip3 install --no-input aqtinstall
      sudo apt update
      sudo apt install -y lsb-release python3-distro
      sudo apt install python3-distro
      pip3 install distro
```
@MNikoliCC  Can we check if these dependencies are necessary? For example, `lsb-release` was present, but the paths were not properly configured.

---

I reviewed the dependencies and removed the unnecessary ones, including `lsb-release`. It should be fine now.